hilti/toolchain/src/ast/ast-context.cc

```diff
@@ -285,6 +353,8 @@ declaration::module::UID ASTContext::_addModuleToAST(declaration::Module* module
     assert(! module->hasParent()); // don't want to end up copying the whole AST
     auto uid = module->uid();
 
+    // We don't need to pin the module for tables because it's tied to the root
+    // node already.
```

Comment:
It isn't really clear to me what is meant here, could you rephrase?

Reply:
That comment is actually stale, no longer applies, removing.